### PR TITLE
[ESPnet2] [WIP] Update CSJ ASR Recipe

### PR DESCRIPTION
**Pull request by YosukeHiguchi**

### CODE REVIEW

**Comment by sw005320** [comment]
This is related to #1490

**Comment by sw005320** [comment]
@YosukeHiguchi, can you check latest changes @kamo-naoyuki made (e.g., some modifications for the bash ci check #1524)?

**Comment by kamo-naoyuki** [comment]
- Would you split this PR to each corpora?
- Please use symbolic links instead of copying for `local/*`.
- If you are ready, notify me.

**Comment by YosukeHiguchi** [comment]
@kamo-naoyuki 

> * Would you split this PR to each corpora?

I'm really sorry but the commits are mixed up with different recipes and it is complicated to split this PR...

**Comment by kamo-naoyuki** [comment]
You dont need  to split commits. it is o.k. to do 
'git rm egs2/foo.'

**Comment by YosukeHiguchi** [comment]
Oh, I see. Thank you! I'll try it.

File: egs2/csj/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- pytorch version: `pytorch 1.0.1.post2`
+- Git hash: `4ea220f8bedac625fded26d03162c12812c15411`
+  - Commit date: `Mon Jan 20 09:45:35 2020 +0900`
+
+## asr_train_rnn_fbank_pitch_char
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_eval1decode_rnn_lm_valid.loss.best_asr_model_valid.acc.best|1272|1272|29.2|70.8|0.0|0.0|70.8|70.8|
```
Comment (kan-bayashi) [inline egs2/csj/asr1/README.md]: WER is the same as S.Err in the case of Japanese.
It is better to remove.

**Comment by YosukeHiguchi** [comment]
@kamo-naoyuki 
I have prepared recipes for CSJ (this PR) and JSUT (#1529 ). Please review these PRs.

File: egs2/csj/asr1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    utils/subset_data_dir.sh --last data/train ${n} data/train_nodev
+
+    # remove duplicated utterances in the training set
+    utils/data/remove_dup_utts.sh 300 data/train_nodev data/${train_set_ori} # 233hr 36min
+
+    # speed purturbation
+    utils/perturb_data_dir_speed.sh 0.9 data/${train_set_ori} data/temp1
+    utils/perturb_data_dir_speed.sh 1.0 data/${train_set_ori} data/temp2
+    utils/perturb_data_dir_speed.sh 1.1 data/${train_set_ori} data/temp3
+    utils/combine_data.sh --extra-files utt2uniq data/${train_set} data/temp1 data/temp2 data/temp3
```
Comment (kamo-naoyuki) [inline egs2/csj/asr1/local/data.sh]: I'll add speed perturbation option in asr.sh, but it's o.k. for now. Just a note.

**Comment by kamo-naoyuki** [comment]
BTW, in my memory, CSJ uses `segments` file, isn't it?
Current version still has bugs about it and it's fixed in #1504.
How did you handle it?

**Comment by YosukeHiguchi** [comment]
@kamo-naoyuki 
Yes, I faced the same problem regrading segment-wise data. For the bug of the usage of `format_wav_scp.sh` in asr.sh, I merged #1513. For the bug in `format_wav_scp.py`, I manually changed my local file.

**Comment by kamo-naoyuki** [comment]
o.k. Thanks.

**Comment by sw005320** [comment]
Thanks.
I'm merging it but the performance seems to be significantly degraded from the previous report.
I think we need to revisit it.

**Comment by kamo-naoyuki** [comment]
I suspect that my LM implementation is something wrong. I didn't test LM enough.

**Pull request merged**